content/2015.09.kinto-encryption-example-fr.rst

```diff
@@ -0,0 +1,254 @@
+Comment chiffrer ses données avec Kinto.js
+##########################################
+
+:date: 2015-09-16
+:summary: Comment utiliser Kinto.js pour chiffrer ses données dans Kinto.
+:slug: kinto-encryption-example
+:lang: fr
+:url: kinto-chiffrer-ses-donnees
+:scripts: kinto-61ad959.min.js, kinto-encryption-example.js
+
+.. note::
+
+    Traduit de l'anglais par `Rémy Hubscher <https://twitter.com/Natim>`_.
+
+Voici un article rédigé par `Michiel de Jong
+<https://michielbdejong.com/>`_ de l'équipe Firefox OS.
+
+Lorsqu'on souhaite sauvegarder les données d'une application
+"Offline-first" (qui fonctionne normalement hors connexion) de manière
+chiffrer dans le cloud, ça a du sens de les garder déchiffrées
+localement et de les chiffrer uniquement avant de les téléverser vers
+un serveur distant.
+
+Ainsi, il est facile et efficace d'y accéder pour les afficher ou
+mettre à jour la copie locale, sans avoir le coût répété de les
+chiffrer/déchiffrer à chaque opération.
+
+De la même manière, on peut déchiffrer les données venant du serveur
+une seule fois, tout de suite après les avoir télécharger et
```

Comment:
Maybe this is the same construction as above, but 
après les avoir télécharger -> après les avoir téléchargé (?)